app/uk/gov/hmrc/thirdpartyapplication/services/ApplicationNamingService.scala

```diff
@@ -67,10 +68,10 @@ abstract class AbstractApplicationNamingService(
     !isValid
   }
 
-  def validateApplicationName(applicationName: String, exclusions: ExclusionCondition): Future[ApplicationNameValidationResult] = {
+  def validateApplicationName(applicationName: ValidatedApplicationName, exclusions: ExclusionCondition): Future[ApplicationNameValidationResult] = {
     for {
-      isDuplicate <- isDuplicateName(applicationName, exclusions)
-      isDenyListed = isDenyListedName(applicationName)
+      isDuplicate <- isDuplicateName(applicationName.value, exclusions)
```

Comment:
CAn you push the type through this call ?

Reply:
Tried this, but these methods are shared by the uplift process where the application can not be guaranteed to be valid because it's the sandbox app name.